jifa-jpackage.sh

```
@@ -0,0 +1,33 @@
+./gradlew clean build -x test
+
```

Should report an error if OS is not macOS.

---

In fact, the jpackage supports common operating systems, but the file name generated after packaging is not very friendly. I'll need to do some optimizations to that script later......